Source/RunActivity/Viewer3D/Shaders.cs

```diff
@@ -255,7 +318,7 @@ public void SetBlurData(ref Matrix wvp)
 
         public void SetBlurData(Texture2D texture)
         {
-            imageTexture.SetValue(texture);
+            blurTexture.SetValue(texture);
```

Comment:
Why do these need to be split out?

Reply:
This change was already applied by gpz, so I'm not able to provide an answer.